package.json

```diff
@@ -11,8 +11,9 @@
   },
   "dependencies": {
     "@babel/runtime": "^7.2.0",
+    "@emotion/core": "^10.0.21",
     "@guardian/atom-renderer": "1.1.1",
-    "@guardian/consent-management-platform": "1.1.0",
+    "@guardian/consent-management-platform": "2.0.0-beta.6",
```

Comment:
We're temporarily using the `beta` build whilst we test, this will be updated.

Reply:
Package version bumped to 2.0.0.